media/css/firefox/family/components/_dad-jokes-banner.scss

```diff
@@ -11,14 +11,16 @@
 
 .c-dad-jokes-banner {
     // related to js show/hide behaviour
-    &[aria-hidden='true'] {
```

Comment:
replaced to due incorrect use of aria-hidden on container with interactive element: https://github.com/mozilla/bedrock/issues/12162